src/data/restrictions.json

```diff
@@ -55,6 +55,18 @@
     {
         "title": "Gospodarka",
         "restrictions": [
+            {
+                "content": "Zamknięcie hoteli"
+            },
+            {
+                "content": "Zamknięcie stoków narciarskich"
+            },
+            {
+                "content": "Zamknięcie sklepów w galeriach handlowych (i sklepów o powierzchni ponad 2000 m kw.) z wyjątkiem sklepów z artykułami:",
+                "details": [
+                    "spożywczymi, kosmetycznymi, środkami czystości, wyrobami medycznymi, remontowo-budowlanymi, dla zwierząt, prasą oraz lokali z usługami"

```

Comment:
jak to wygląda na stronie?

Reply:
to dosłownie jest z poprzedniego commita, wygląda ok imo